packages/runtime/container-runtime/src/summaryFormat.ts

```diff
@@ -96,6 +91,25 @@ export interface ICreateContainerMetadata {
     createContainerTimestamp?: number;
 }
 
+export type GCVersion = number;
+export interface IGCMetadata {
+    /**
+     * The version of the GC code that was run to generate the GC data that is written in the summary.
+     * Also, used to determine whether GC is enabled for this container or not:
+     * - A value of 0 or undefined means GC is disabled.
+     * - A value > 0 means GC is enabled.
+     */
+    readonly gcFeature?: GCVersion;
+    /** If this is present, the session for this container will expire after this time and the container will close */
+    readonly sessionExpiryTimeoutMs?: number;
+    /**
+     * Tells whether the GC sweep phase is enabled for this container.
+     * - True means sweep phase is enabled.
+     * - False means sweep phase is disabled. If GC is disabled as per gcFeature, sweep is also disabled.
+     */
+    readonly sweepEnabled?: boolean;
```

Comment:
Same as above, maybe `sweepAllowed`, so that it sounds like `shouldRunGC` is the final say, and the RunSweep flag overrides it